extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php

```diff
@@ -213,6 +218,7 @@ public static function isPageBlockedViaApi($amgId="", $gracenoteId="", $pageTitl
 					$isBlocked = true;
 					break;
 
+				case self::CODE_SUCCESS_NO_LYRICS:
```

Comment:
I looked into it... I don't think it would really add anything in this case (we don't exhaustively test every valid & invalid status code & it would be hard to do it right without knowing all of the other codes... since any other bugs would just presumably be undocumented codes: and we'd be writing a test to say what we think the code should return, without knowing what it actually should return).